main/lsp/requests/completion.cc

```diff
@@ -521,6 +521,33 @@ unique_ptr<CompletionItem> getCompletionItemForConstant(const core::GlobalState
     return item;
 }
 
+unique_ptr<CompletionItem> getCompletionItemForKwarg(const core::GlobalState &gs, const LSPConfiguration &config,
+                                                     const core::NameRef local, const core::Loc queryLoc,
+                                                     string_view prefix, size_t sortIdx) {
+    auto label = local.shortName(gs);
+    auto item = make_unique<CompletionItem>(absl::StrCat(label, ": (keyword argument)"));
+    item->sortText = formatSortIndex(sortIdx);
+    item->kind = CompletionItemKind::Keyword;
```

Comment:
I'm not sure what the right completion kind here is. `Keyword` is pretty clearly not right, but I'm not sure what a better option is.